lib/chat_app/contexts/messages.ex

```diff
@@ -0,0 +1,22 @@
+defmodule ChatApp.Contexts.Messages do
+  alias ChatApp.Structs.Message
+  alias ChatApp.Repo
+
+  def list_messages(), do: Repo.all(Message)
```

Comment:
list by room Id, nu?

Reply:
Nu stiu daca mai e nevoie, avand in vedere ca list_rooms si get_room(id) arata mesajele oricum.

Daca e nevoie, adaug si aici o functie

Reply:
get_room si get_room_messages pot fi 2 functii separate. una sa iti dea info despre room, alta despre mesaje